components/brave_rewards/resources/ui/reducers/wallet_reducer.ts

```diff
@@ -219,7 +219,11 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
           ...state,
           ui
         }
-
+      }
```

Comment:
need a `break` in here somewhere for lint

Reply:
fixed